docs/source/getting-started/content/proc_accessing-the-openshift-cluster.adoc

```diff
@@ -12,13 +12,22 @@ For more information, see <<starting-the-virtual-machine_{context}>>.
 
   . Run [command]`{bin} console`. This will open your web browser and direct it to the web console.
 
-  . Log in to the OpenShift web console as the `kubeadmin` user with the password printed in the output of the [command]`{bin} start` command.
+  . Log in to the OpenShift web console as the `developer` user with the password printed in the output of the [command]`{bin} start` command.
```

Comment:
Should we mention selecting `htpasswd_provider` in the web console? I believe this has confused users in the past.

Reply:
Yes we should mention that for web console, it really confusing atm.

Reply:
I've filed https://github.com/code-ready/crc/issues/679